apps/frontend/src/components/ReOrganizer/ReOrganizer.tsx

```diff
@@ -7,6 +7,8 @@ import { selectPreviewTree, updateTree } from '@store/previewTree/previewTreeSli
 import SideBarSearchBar from '@components/SideBar/SideBarSearchBar';
 import SideBarTabTitle from '@components/SideBar/SideBarTabTitle';
 import ReOrganizerItem from './ReOrganizerItem';
+import Contextable from '../Decorators/Contexable';
```

Comment:
⚠️  Use the defined import aliases ! here `@components/Decorators/Contexable`

Reply:
Ok